.github/workflows/build.yml

```diff
@@ -259,23 +258,14 @@ jobs:
         uses: docker/setup-qemu-action@v3
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
-      - name: Cache Docker layers
-        uses: actions/cache@v4
-        env:
-          BASE_CACHE_KEY: buildx-${{ runner.os }}-
-        with:
-          key: ${{ env.BASE_CACHE_KEY }}${{ github.sha }}
-          path: ${{ env.BUILDX_CACHE_DIR }}
-          restore-keys: |
-            ${{ env.BASE_CACHE_KEY }}
       - name: Create dist directory
         run: mkdir -p dist
       - name: Build image
         id: docker_build
         uses: docker/build-push-action@v6
         with:
-          cache-from: type=local,src=${{ env.BUILDX_CACHE_DIR }}
-          cache-to: type=local,dest=${{ env.BUILDX_CACHE_DIR }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
```

Comment:
What do you think about adding a brief comment explaining why we prefer the `max` cache mode to `min`?

Reply:
Please let me know what you think of https://github.com/cisagov/skeleton-docker/pull/222/commits/677e97f725aabcf1decd67f6b588dfda75037e7c.

Reply:
Perfect!

Reply:
Excellent, thanks!